pkg/kubernetes/views/deploymentconfig-panel.html

```diff
@@ -5,7 +5,7 @@
             <i class="fa fa-gear"></i>
         </div>
         <div class="listing-actions">
-            <button title="Delete" class="btn btn-danger btn-delete pficon pficon-delete" disabled></button>
+            <button id="delete-deplconfig" title="Delete" class="btn btn-danger btn-delete pficon pficon-delete delete-entity " data-toggle="modal" data-target="#delete-entity-dialog" data-link="{{ item.metadata.selfLink }}" data-key="{{item.key}}"></button>
```

Comment:
This should not have a an ```id='delete-deplconfig'```. This template is instantiated multiple times per page, and that would result in an invalid page.